oeps/oep-0010.rst

```diff
@@ -0,0 +1,202 @@
+=========================
+OEP-10: Open edX Releases
+=========================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`OEP-10 <oep-0010>`                  |
++---------------+-------------------------------------------+
+| Title         | Open edX Releases                         |
++---------------+-------------------------------------------+
+| Last Modified | 2016-10-14                                |
++---------------+-------------------------------------------+
+| Author        | Ned Batchelder, ned@edx.org               |
++---------------+-------------------------------------------+
+| Arbiter       | Edward Fagin                              |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Process                                   |
++---------------+-------------------------------------------+
+| Created       | 2016-10-14                                |
++---------------+-------------------------------------------+
+| `Resolution`  | <links to any discussions where the final |
+|               | status was decided>                       |
++---------------+-------------------------------------------+
+
+
+..
+    - Expectations for component owners
+
+
+Abstract
+========
+
+Open edX is packaged occasionally into releases. This document details the
+process.
+
+
+Motivation
+==========
+
+Open edX releases happen infrequently, every four to six months. Edx engineers
```

Comment:
Would it be a good place to discuss switching to a different release schedule? As mentioned in other places, it would be useful for this to be predictible, so that upgrades can be scheduled once for all, and make it easier for course teams of Open edX instances to plan their courses schedules around this?